app/src/test/java/org/simple/clinic/bp/entry/BloodPressureEntrySheetControllerTest.kt

```diff
@@ -316,15 +313,15 @@ class BloodPressureEntrySheetControllerTest {
   @Suppress("IMPLICIT_CAST_TO_ANY")
   @Parameters(method = "params for checking valid date input")
   fun `when save is clicked, date entry is active, but input is invalid then BP measurement should not be saved`(
-      openAs: OpenAs,
-      result: UserInputDateValidator.Result
+      testParams: DoNotSaveBpWithInvalidDateTestParams
   ) {
+    val (openAs, result) = testParams
```

Comment:
The `result` here is unused. Dod you mean to add a verification that the date validator returns this result?

Reply:
This can go, because the assertion has changed.

Reply:
Fixed.